### PR TITLE
Run kuttl tests in e2e job for ironic-operator

### PR DESCRIPTION
**Pull request by cescgina**
Run the build-deploy-kuttl workflow instead of the build-deploy one.

### CODE REVIEW

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-keystone-operator-build-deploy-kuttl

File: ci-operator/config/openstack-k8s-operators/ironic-operator/openstack-k8s-operators-ironic-operator-master.yaml [hunk 1]
```diff
@@ -74,6 +86,17 @@ tests:
   optional: true
   steps:
     workflow: openstack-k8s-operators-test-build-deploy
+- as: keystone-operator-build-deploy-kuttl
```
Comment (viroel) [inline ci-operator/config/openstack-k8s-operators/ironic-operator/openstack-k8s-operators-ironic-operator-master.yaml:89]: ironic

**Review by viroel — CHANGES_REQUESTED** [review]

File: ci-operator/config/openstack-k8s-operators/ironic-operator/openstack-k8s-operators-ironic-operator-master.yaml (hunk 1 above)
Comment (cescgina) [reply ci-operator/config/openstack-k8s-operators/ironic-operator/openstack-k8s-operators-ironic-operator-master.yaml]: Fixed, thanks!

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-kuttl

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-kuttl

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-kuttl

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-kuttl

**Comment by Sandeepyadav93** [comment]
/pj-rehearse ack

kuttl run looks good

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/37103/rehearse-37103-pull-ci-openstack-k8s-operators-ironic-operator-master-ironic-operator-build-deploy-kuttl/1637751620852781056/artifacts/ironic-operator-build-deploy-kuttl/openstack-k8s-operators-kuttl/build-log.txt

**Pull request merged**